site/public/css/course-materials.css

```diff
@@ -44,3 +44,7 @@ a.popout-item > .fas:hover {
 .delete-course-material-message {
     font-weight: bold;
 }
+
+a.btn-primary {
+    margin: 1.5px;
+}
```

Comment:
This is going to target *all* buttons inside links.  That's probably not a good thing if we have a button somewhere else.  It would be better to give these buttons a class and target that class with CSS.